IdentityCore/src/telemetry/MSIDTelemetry+Cache.m

```diff
@@ -33,8 +35,7 @@ @implementation MSIDTelemetry (Cache)
 + (MSIDTelemetryCacheEvent *)startCacheEventWithName:(NSString *)cacheEventName
                                              context:(id<MSIDRequestContext>)context
 {
-    [[MSIDTelemetry sharedInstance] startEvent:[context telemetryRequestId]
-                                     eventName:cacheEventName];
+    CONDITIONAL_START_EVENT(CONDITIONAL_SHARED_INSTANCE, [context telemetryRequestId], cacheEventName);
```

Comment:
This entire class is excluded, so changes in this file are not needed

Reply:
In production code, all the reference of startEvent and stopEvent are wrapped by these MACROs, I prefer to keep the consistency.